test/hotspot/jtreg/runtime/TLS/exestack-tls.c

```diff
@@ -54,6 +57,7 @@ JNIEnv* create_vm(JavaVM **jvm, char* argTLS) {
     return env;
 }
 
+#if defined(__GLIBC)
```

Comment:
Why do we use this form here but at line 30 we have:
#ifdef __GLIBC__
?

Reply:
Fixed to be consistent.